database/README.md

```diff
@@ -0,0 +1,168 @@
+# Database instructions
+
+## Requirements
+
+* Kubernetes cluster
+  
+  Follow the instructions to deploy a Kubernetes cluster from [Local Deployment]([kubernetes](https://kubernetes.io/docs/tasks/tools/#kubectl)).
```

Comment:
This link is not displaying at all when I view the file in GitHub. Was the link intended to be for the "Local Deployment" text?

Reply:
updated!